extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/vertx/HttpInstrumenterVertxTracer.java

```diff
@@ -231,12 +232,12 @@ public Long requestContentLengthUncompressed(final HttpRequest request, final Ht
 
         @Override
         public Integer statusCode(final HttpRequest request, final HttpResponse response) {
-            return response.statusCode();
+            return response != null ? response.statusCode() : 499;
```

Comment:
Please, return `null` here if the `response` is null. The extractor can handle such return value.

Reply:
Ah yes, I thought it made sense but apparently `499` is specific to Nginx and is not standard.